packages/node_modules/@webex/plugin-meetings/src/member/index.js

```diff
@@ -1,197 +0,0 @@
-/*!
- * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
- */
-
-/**
- * Events to emit
- * ==============
- *
- * - change (string): audioMuted | videoMuted | left | joined | madeHost | shared
- * // NOTE: /calls API had: waiting | lobby | notified | declined | joined | left
- */
-
-/* eslint-disable */
-export default class Member {
-  /**
-   * Create a Member.
-   * @param {string} id
-   */
-  constructor({id, isGuest, isHost, isAudioMuted, isVideoMuted, devices}) {
-    this.id = id;
-    // TODO: Make this thenable?
-    this.isGuest = isGuest;
```

Comment:
Are "members" changing to "participants"? Will we be able to distinguish between people *participating* in the meeting (attending) vs. those just on the meeting invite? Will participants have these same characteristics, e.g., isHost, etc.?